package.json

```diff
@@ -132,7 +132,8 @@
     "react-dropzone": "^10.2.1",
     "react-rte": "^0.16.1",
     "recharts": "^1.8.5",
-    "whatwg-fetch": "^3.0.0"
+    "whatwg-fetch": "^3.0.0",
+    "wp-prettier": "git+https://github.com/Automattic/wp-prettier.git#wp-prettier-1.18.2"
```

Comment:
Why is this here? We're [already using `wp-prettier` as `prettier`](https://github.com/Automattic/newspack-plugin/blob/82fc7fddd423021bb22238167d0f6fc54d08813a/package.json#L49)

Reply:
🤔  no idea, I simply ran `npm update`. Should I just remove the line?

Reply:
Confirmed, that's probably added because of that `prettier` aliasing 🤷‍♀️